Base/usr/share/man/man8/TelnetServer.md

```diff
@@ -4,22 +4,23 @@ TelnetServer - Serenity telnet server
 
 ## Synopsis
 
-```**sh
-# TelnetServer [options]
+```sh
+$ TelnetServer [-p port] [-c command]
```

Comment:
I doubt this runs as a non-root user.

Reply:
You are correct. That should be added to the man page.